3.2 additions (#286)
mathjax/MathJax-docs: options/startup/loader.rst

   data object as an argument, or an array consisting of such a
   function and a number representing its priority in the list of
   filters (lower numbers are earlier in the list).  The data object
   that is passed to these functions is

   .. code-block:: javascript

      {
        name: string,            // the current name for the package (this becomes the url in the end)
        original: string,        // the original package name (should not be modified)
        addExtension: boolean,   // true if .js should be added to this name at some stage in the filter list
      }

   The filter can change the `name` value to move it closer to the
   final URL used for loading the given package.  The `original`
   property should be the original name of the package, and should not
   be modified.

   The function should return ``true`` if the `name` should be futher
   processed by other filters in the list, and ``false`` to end
   processing with the `name` now representing the final URL for the
   component.

   There are three default filters: one that replaces `name` with its
   value in the ``source`` list, if any; one that normalizes package

>    names by adding ``[mathjax]/`` if there isn't a prefix or protocol

Reviewer: ```suggestion
   names by adding ``[mathjax]/`` if there is no prefix or protocol
```

>    already, and adding ``.js`` if there isn't an extension; and one

Reviewer: ```suggestion
   already, and adding ``.js`` if there is no extension; and one
```
